### PR TITLE
changed content in line with GIT recommendations

### PR DESCRIPTION
**Pull request by MaeveRoseveare**
Get Into Teaching have asked us to update some content and links from various places on the prototype.

Trello ticket: https://trello.com/c/06Nbw6Sg/1011-go-through-and-add-link-from-apply-to-git

### CODE REVIEW

**Comment by MaeveRoseveare** [comment]
[Go through and add link from Apply to GIT](https://trello.com/c/06Nbw6Sg/1011-go-through-and-add-link-from-apply-to-git)

File: app/views/application/choices/find.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 {% block primary %}
   <p class="govuk-body-l">Search for courses near you on Find postgraduate teacher&nbsp;training.</p>
+  <p class="govuk-body-l">A <a href="https://adviser-getintoteaching.education.gov.uk/">teacher training adviser</a> can help you understand your course options and support you with your application.</p>
```
Comment (frankieroberto) [inline app/views/application/choices/find.html:14]: Does this sentence still look ok using the larger font (`govuk-body-l`) or should we drop it down to regular size?

File: app/views/application/additional-support/index.html [hunk 2]
```diff
@@ -38,12 +38,12 @@
...
 
   <h2 class="govuk-heading-m">It’s against the law to discriminate</h2>
-  <p class="govuk-body">If you’re disabled, <a href="https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability">it’s against the law to discriminate against you</a>. Training providers must not:</p>
+  <p class="govuk-body">If you’re disabled, it’s against the law to discriminate against you. Training providers must not:</p>
   <ul class="govuk-list govuk-list--bullet">
     <li>ask disability or health questions if they’re not relevant to your ability to become a teacher</li>
     <li>reject your application because you’re disabled</li>
   </ul>
-
+  <p class="govuk-body"><a href="https://getintoteaching.education.gov.uk/funding-and-support/if-youre-disabled">Get support training to teach if you're disabled.</a></p>
```
Comment (frankieroberto) [inline app/views/application/additional-support/index.html:46]:
```suggestion
  <p class="govuk-body"><a href="https://getintoteaching.education.gov.uk/funding-and-support/if-youre-disabled">Get support training to teach if you’re disabled.</a></p>
```

File: app/views/application/decision/view.html [hunk 3]
```diff
@@ -27,7 +27,7 @@ <h1 class="govuk-heading-l">{{ title }}</h1>
         <li>if you decline all of your offers, or do not respond in time, you can still apply for courses that start this academic year</li>
       </ul>
 
-      <p class="govuk-body">If you need help, you can speak to our <a href="https://beta-adviser-getintoteaching.education.gov.uk" class="govuk-link">teacher training advisers</a>.</p>
+      <p class="govuk-body">If you need help, you can speak to our <a href="https://beta-adviser-getintoteaching.education.gov.uk" class="govuk-link">teacher training advisers</a>. They're all experienced former teachers who can give you free, one-to-one support.</p>
```
Comment (frankieroberto) [inline app/views/application/decision/view.html:30]:
```suggestion
      <p class="govuk-body">If you need help, you can speak to our <a href="https://beta-adviser-getintoteaching.education.gov.uk" class="govuk-link">teacher training advisers</a>. They’re all experienced former teachers who can give you free, one-to-one support.</p>
```

File: app/views/application/subject-knowledge/index.html [hunk 4]
```diff
@@ -33,6 +33,7 @@
     <li>your understanding of the national curriculum</li>
   </ul>
 
+  <p class="govuk-body">If you need help, you can speak to our <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers">teacher training advisers</a>. They're all experienced former teachers who can give you free, one-to-one support with your personal statement. </p>
```
Comment (frankieroberto) [inline app/views/application/subject-knowledge/index.html:36]:
```suggestion
  <p class="govuk-body">If you need help, you can speak to our <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers">teacher training advisers</a>. They’re all experienced former teachers who can give you free, one-to-one support with your personal statement. </p>
```

File: app/views/dashboard/index.html [hunk 5]
```diff
@@ -70,13 +70,13 @@ <h2 class="govuk-heading-m">
           <p class="govuk-body">You will be emailed when {% if choices | length > 1 %}a provider{% else %}the provider{% endif %} makes a decision.</p>
 
 
-          <p class="govuk-body">Our <a href="https://beta-adviser-getintoteaching.education.gov.uk/">teacher training advisers</a> can help you prepare for any interviews.</p>
+          <p class="govuk-body">Our <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers">teacher training advisers</a> can help you prepare for any interviews. All our advisers are experienced former teachers who can give you free, one-to-one support and help you as much or as little as you need.</p>
```
Comment (frankieroberto) [inline app/views/dashboard/index.html:73]: Cut this down a bit?

```suggestion
          <p class="govuk-body">Our <a href="https://getintoteaching.education.gov.uk/teacher-training-advisers">teacher training advisers</a> can help you prepare for any interviews. Our advisers are former teachers who can give you free, one-to-one support.</p>
```

**Review by frankieroberto — APPROVED** [review]

**Review by frankieroberto — APPROVED** [review]

File: app/views/application/choices/find.html (hunk 1 above)
Comment (MaeveRoseveare) [reply app/views/application/choices/find.html:14]: Good shout, looks much better as just body font!

**Pull request merged**